apps/condo/domains/miniapp/components/AppDescription/AppDescriptionPageContent.tsx

```diff
@@ -46,6 +48,31 @@ export const AppDescriptionPageContent: React.FC<AppDescriptionPageContentProps>
         buttonLabel: ConnectButtonLabel,
     })
 
+    const { logEvent, eventProperties, getEventName } = useTracking()
+
+    useEffect(() => {
+        const routeChangeStart = (event: MouseEvent) => {
+            const target = event.target as HTMLElement
+            if (target.tagName === 'A') {
+                const value = get(target, 'attributes.href.value', '')
+                const title = get(target, 'textContent', '')
+                const eventName = getEventName(TrackingEventType.FollowExternalLink)
+
+                logEvent({ eventName, eventProperties: {
+                    ...eventProperties,
+                    link: { title, value },
+                } })
+            }
+        }
+        if (typeof window !== 'undefined') {
+            document.addEventListener('click', routeChangeStart)
+        }
+        return () => {
+            document.removeEventListener('click', routeChangeStart)
+        }
```

Comment:
looks little hacky, con we use it globally?

Reply:
@pahaz I don't think so, cause it page related logic